app/Resources/translations/validators.de.xliff

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="de" datatype="plaintext" original="file.ext">
+        <body>
+        <trans-unit id="post.blank_summary">
+            <source>post.blank_summary</source>
+            <target>Gib deinem Beitrag eine Zusammenfassung!</target>
+        </trans-unit>
+        <trans-unit id="post.too_short_content">
+            <source>post.too_short_content</source>
+            <target>Der Beitragsinhalt ist zu kurz (mindestens {{ limit }} Zeichen)</target>
```

Comment:
There are no exclamation marks in the english/french/... translation file as well.